cds_dojson/schemas/records/books/ymls/series-v.0.0.1.yml

```diff
@@ -0,0 +1,23 @@
+$schema: 'http://json-schema.org/schema#'
```

Comment:
since these are new schemas, do you need the `yml`?

Reply:
I'm just generating everything from .ymls anyway and it is faster to write yml schema. In general we don't really need ymls